libdnf/sack/packageset.cpp

```diff
@@ -97,12 +97,61 @@ PackageSet::operator +=(const PackageSet & other)
     return *this;
 }
 
+PackageSet &
+PackageSet::operator -=(const PackageSet & other)
+{
+    map_subtract(&pImpl->map, &other.pImpl->map);
+    return *this;
+}
+
+PackageSet &
+PackageSet::operator /=(const PackageSet & other)
+{
+    map_and(&pImpl->map, &other.pImpl->map);
+    return *this;
+}
+
+PackageSet &
+PackageSet::operator +=(const Map * other)
+{
+    map_or(&pImpl->map, other);
+    return *this;
+}
+
+PackageSet &
+PackageSet::operator -=(const Map * other)
+{
+    map_subtract(&pImpl->map, other);
```

Comment:
/libdnf/libdnf/sack/packageset.cpp: In member function ‘libdnf::PackageSet& libdnf::PackageSet::operator-=(const Map*)’:
/home/rfonseca/prjs/upstream/libdnf/libdnf/sack/packageset.cpp:124:31: error: invalid conversion from ‘const Map*’ {aka ‘const _Map*’} to ‘Map*’ {aka ‘_Map*’} [-fpermissive]
map_subtract(&pImpl->map, other);
^~~~~